packages/arb-token-bridge-ui/synpress.cctp.config.ts

```diff
@@ -0,0 +1,135 @@
+import { Wallet, utils } from 'ethers'
+import { defineConfig } from 'cypress'
+import { StaticJsonRpcProvider } from '@ethersproject/providers'
+import synpressPlugins from '@synthetixio/synpress/plugins'
+import logsPrinter from 'cypress-terminal-report/src/installLogsPrinter'
+import { getCommonSynpressConfig } from './tests/e2e/getCommonSynpressConfig'
+import {
+  setupCypressTasks,
+  fundEth,
+  fundUsdc,
+  getCustomDestinationAddress
+} from './tests/support/common'
+import specFiles from './tests/e2e/cctp.json'
+import { sepolia } from 'wagmi'
+
+const shouldRecordVideo = process.env.CYPRESS_RECORD_VIDEO === 'true'
+
+const tests = process.env.TEST_FILE
+  ? [process.env.TEST_FILE]
+  : specFiles.map(file => file.file)
+
+const INFURA_KEY = process.env.NEXT_PUBLIC_INFURA_KEY
+if (typeof INFURA_KEY === 'undefined') {
+  throw new Error('Infura API key not provided')
+}
+
+const SEPOLIA_INFURA_RPC_URL = `https://sepolia.infura.io/v3/${INFURA_KEY}`
+const ARB_SEPOLIA_INFURA_RPC_URL = `https://arbitrum-sepolia.infura.io/v3/${INFURA_KEY}`
```

Comment:
we are using these in `synpress.config.ts`

```typescript
const sepoliaRpcUrl =
  process.env.NEXT_PUBLIC_SEPOLIA_RPC_URL ?? SEPOLIA_INFURA_RPC_URL
const arbSepoliaRpcUrl = 'https://sepolia-rollup.arbitrum.io/rpc'
```

why don't we use the same in this file?